index.html

```diff
@@ -23,9 +23,23 @@
     <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://www.gstatic.com/firebasejs/4.3.1/firebase.js"></script>
     <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
     <script type="text/javascript" src="lib/chosen/chosen.jquery.min.js" ></script>
 
+    <script>
+      var config = {
+        apiKey: "AIzaSyDrpK7dIL7uCqyI8mC7wP1JwD42lbkOLHk",
```

Comment:
Esta API Key debe de ser destruida y la nueva no ser puesta en el repositorio.

Referencia:
- Do not embed API keys directly in code:
- Do not store API keys in files inside your application's source tree
[Best practices for securely using API keys
](https://support.google.com/cloud/answer/6310037)

Reply:
utilizan GRUNT.JS en el back-end o cual??? asi evitan mostrar la API atraves de un npm para keys si necesita ayuda en el back-end o front-end puedo ayudar saludos

Reply:
La apiKey sólo tiene permisos de lectura, entonces no hay mucho problema.